assets/css/main.scss

```diff
@@ -4,6 +4,41 @@
 @charset "UTF-8";
 
 @import "identity-style-guide/dist/assets/scss/styles";
+
+// START: TODO: Remove and reconcile after upgrading to USWDS >= 2.9.0
```

Comment:
Yep, it's a grotesque hack to get identifier from USWDS here. And yes, the grotesqueness is intentionally on full display to motivate future cleanup.